modal/sandbox.py

```diff
@@ -177,6 +183,9 @@ def _new(
         cloud_bucket_mounts = [(k, v) for k, v in validated_volumes if isinstance(v, _CloudBucketMount)]
         validated_volumes = [(k, v) for k, v in validated_volumes if isinstance(v, _Volume)]
 
+        if pty:
+            pty_info = _Sandbox._default_pty_info()
```

Comment:
That means we are using the parameters from the local shell for pty. I'm okay with that.

Reply:
Yes exactly, if they are available, and else they are left as absent.